docs/content/getting-started/logging-cpp.md

```diff
@@ -2,5 +2,379 @@
+
+```bash
+cmake -B build
+cmake --build build -j
+./build/example_dna
+```
+
+When everything finishes compiling, an empty Rerun viewer should be spawned:
+<picture>
+  <img src="https://static.rerun.io/rerun-welcome-screen-0.9/cc45a0700ccf02016fb942153106db4af0c224db/full.png" alt="">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/rerun-welcome-screen-0.9/cc45a0700ccf02016fb942153106db4af0c224db/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/rerun-welcome-screen-0.9/cc45a0700ccf02016fb942153106db4af0c224db/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/rerun-welcome-screen-0.9/cc45a0700ccf02016fb942153106db4af0c224db/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/rerun-welcome-screen-0.9/cc45a0700ccf02016fb942153106db4af0c224db/1200w.png">
+</picture>
+
+## Logging our first points
+
+Now let's add some data to the viewer.
+
+The core structure of our DNA looking shape can easily be described using two point clouds shaped like spirals.
+Add the following to your `main` function:
+```cpp
+static constexpr size_t NUM_POINTS = 100;
+
+std::vector<rerun::Position3D> points1, points2;
+std::vector<rerun::Color> colors1, colors2;
+color_spiral(NUM_POINTS, 2.0f, 0.02f, 0.0f, 0.1f, points1, colors1);
+color_spiral(NUM_POINTS, 2.0f, 0.02f, TAU * 0.5f, 0.1f, points2, colors2);
+
+rec.log(
+    "dna/structure/left",
+    rerun::Points3D(points1).with_colors(colors1).with_radii({0.08f})
+);
+rec.log(
+    "dna/structure/right",
+    rerun::Points3D(points2).with_colors(colors2).with_radii({0.08f})
+);
+```
+
+Re-compile and run your program again:
+```bash
+cmake --build build -j
+./build/example_dna
+```
+and now you should now see this scene in the viewer:
+
+<picture>
+  <img src="https://static.rerun.io/logging_data3_first_points/95c9c556160159eb2e47fb160ced89c899f2fcef/full.png" alt="">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/logging_data3_first_points/95c9c556160159eb2e47fb160ced89c899f2fcef/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/logging_data3_first_points/95c9c556160159eb2e47fb160ced89c899f2fcef/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/logging_data3_first_points/95c9c556160159eb2e47fb160ced89c899f2fcef/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/logging_data3_first_points/95c9c556160159eb2e47fb160ced89c899f2fcef/1200w.png">
+</picture>
+
+_This is a good time to make yourself familiar with the viewer: try interacting with the scene and exploring the different menus._
+_Checkout the [Viewer Walkthrough](viewer-walkthrough.md) and [viewer reference](../reference/viewer/overview.md) for a complete tour of the viewer's capabilities._
+
+### Under the hood
+
+This tiny snippet of code actually holds much more than meets the eye…
+
+`Archetypes`
```

Comment:
You should use proper heading for this and the other… well, headings :)

Looks better, reads better, works with screen-readers, etc